operator/build/Dockerfile.ansible-s390x

```diff
@@ -0,0 +1,31 @@
+#FROM quay.io/operator-framework/ansible-operator:v0.11.0
+FROM quay.io/mew2057/ansible-operator:latest
```

Comment:
Oh, I see,  I think you build this custom one right, we have to use this one to get this working for now? 

Would it make any sense to put some of these images to the ibm-spectrum-scale-csi-dev org on quay.io?

Reply:
It might, I have it on mine  for now, because I don't necessarily want this to be  any kind of long  term.